### ISSUE TITLE
Down / up arrow key navigation not working in Select component when VoiceOver enabled

### ISSUE THREAD

**Issue by frederickf** [adobe/react-spectrum#6256]
### Provide a general summary of the issue here

When VoiceOver is enabled while using Safari or Firefox, the down and up arrow keys don't work in the Select component. Instead it just closes.

This happening in the project I'm working on and I can reproduce it in the official docs as described in the Reproduce section.

To be clear, it only happens when VoiceOver is enabled. Without VoiceOver keyboard navigation works fine. It also works brilliantly in Chrome, with and without VoiceOver

Thanks in advance

### 🤔 Expected Behavior?

When VoiceOver is enabled, the down and up arrow keys should navigate the items in the list

### 😯 Current Behavior

When VoiceOver is enabled, the down and up arrow keys are causing the Select to close

### 🖥️ Steps to Reproduce

1. Use Safari https://react-spectrum.adobe.com/react-aria/Select.html
2. Enable VoiceOver
3. Focus the first select component on the page. The one with animal names.
4. Press enter to open the listbox
5. Press the down arrow key

For me after step 5 the list just closes

### Version

1.1.1

### What browsers are you seeing the problem on?

Firefox, Safari

### What operating system are you using?

Mac OS Sonoma 14.4

**Comment by LFDanLu** [comment]
The team has noticed something similar with table/tree navigation recently with Sonoma 14.4 as well, I don't see the issue you described in Sonoma 14.3 so maybe something changed in VoiceOver with this update. I'll update you after I bump my own machine's OS

**Comment by LFDanLu** [comment]
Odd I can't reproduce, I'm now on Sonoma 14.4.1, Safari 17.4.1. @snowystinger are you able to?

**Comment by frederickf** [comment]
@LFDanLu  Thanks for the quick response. I have a few updates.

I realized my description isn't quite accurate. I described the behavior for Firefox. In Safari, pressing the down arrow with VoiceOver enabled move the focus to the hidden Dismiss button.

I saw that you were on Sonoma 14.4.1, so tried updating and that didn't help either. But since you said it is working for you I did some more digging and figured out my problem.         

Apparently I had [Quick Nav](https://support.apple.com/guide/voiceover/with-quick-nav-vo27943/mac ) enabled. I'd never heard of it before until my googling led me to this [reddit thread](https://www.reddit.com/r/Blind/comments/ssuew9/arrow_keys_not_working_in_word_when_using/). I must have enabled it accidentally. I disabled it and arrow keys work as expected now.

I'm going to go ahead and close this. Thanks again
